README.md

```diff
@@ -29,10 +29,6 @@ Main features
 - :microscope: Tiny and tree-shakeable
 - :muscle: Supports TypeScript
 
-## Examples
-
-- [This is a contrived example](https://codesandbox.io/s/crazy-wood-vn7gg?file=/src/fakeApi.js) based on [this example](https://reactjs.org/docs/concurrent-mode-patterns.html#reviewing-the-changes) from the React docs.
-
 ## Docs
 ```tsx
 const [useDocs] = connectObservable(NEVER)
```

Comment:
Now that I think of, ☝️ this kinda reads as docs will NEVER come :P

Reply:
Yep... I'm not sure if ppl will find that funny :grimacing: So, we better hurry up and finish the first version of the docs.